www/src/content/blog/2024-12-03-cf-service-bindings.mdx

```diff
@@ -36,7 +36,7 @@ Separately, and not as part of the registration process, you offer a newsletter
 When an athlete subscribes to the newsletter, their email address is stored in a different database, and they receive a welcome email.
 
 In this scenario, you could have three workers: one for handling the sign-up form, another for managing the newsletter subscription, and then a _shared_ email worker to send emails.
-![Example Workers architecture](@/assets/blog/2024-12-03-workers-example-architecture.png)
+![Example Workers architecture](@/assets/blog/2024-12-03-queue-architecture.png)
```

Comment:
not sure you wanted to change this image? the queue architecture image doesn't seem to match the description of the preceding sentence